doozer/doozerlib/cli/images_streams.py

```diff
@@ -831,6 +832,25 @@ def search_issues(query):
             new_issues[distgit_key] = issue
             print(f'A JIRA issue has been opened for {pr.html_url}: {issue.key}')
             connect_issue_with_pr(pr, issue.key)
+            try:
+                # Check if the custom fields are already set
+                release_notes_text_value = issue.fields.__dict__.get('customfield_12317313')
+                release_notes_type_value = issue.fields.__dict__.get('customfield_12320850')
+
+                if release_notes_type_value not in ['Release Notes Not Required'] and release_notes_text_value not in ['N/A']:
```

Comment:
This would overwrite human actions in case humans would update this. Think this bot should do:
If it is set at all: Do not touch.
If it is empty: Initialize it.

Reply:
Now it only updates this fields when both are empty